code/game/machinery/vendors/wardrobe_vendors.dm

```diff
@@ -100,26 +100,26 @@
 
 	vend_reply = "Thank you for using the DetDrobe!"
 	products = list(/obj/item/clothing/under/rank/security/detective = 2,
-					/obj/item/clothing/under/rank/security/detective/black = 1,
-					/obj/item/clothing/under/rank/security/detective/skirt = 1,
+					/obj/item/clothing/under/rank/security/detective/black = 2,
```

Comment:
This is the detective vendor (which costs money). You said locker in the CI so you may wish to fix that

Reply:
adding it to the CL, intentional change as I said in description